docs/guides/hello-world.ipynb

```diff
@@ -180,7 +180,7 @@
     {
```

Comment:
<p>@kaelynj does this new data look ok?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/documentation/pull/1709/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
<p>I'd try running it again. Looking at the previous data, the spread of expectation values is quite wide here and doesn't demonstrate the decreasing &lt;Z_0Z_i&gt; as nicely.</p>

Reply:
<p>After offline discussion, we'll remove this notebook from the PR as these changes don't add anything to improve the user experience, and the old data better demonstrates the ideas.</p>